webhook/resources/volume/mutator.go

```diff
@@ -303,6 +303,12 @@ func (v *volumeMutator) Update(request *admission.Request, oldObj runtime.Object
 	if string(volume.Spec.BackendStoreDriver) == "" {
 		patchOps = append(patchOps, fmt.Sprintf(`{"op": "replace", "path": "/spec/backendStoreDriver", "value": "%s"}`, longhorn.BackendStoreDriverTypeV1))
 	}
+	if string(volume.Spec.BackupCompressionMethod) == "" {
+		patchOps = append(patchOps, fmt.Sprintf(`{"op": "replace", "path": "/spec/backupCompressionMethod", "value": "%s"}`, longhorn.BackupCompressionMethodGzip))
```

Comment:
If not set, isn't it `BackupCompressionMethodNone` instead?

Reply:
No, this is for the case that an existing volume with empty BackupCompressionMethod. The compression method is gzip.

Reply:
OH I see, thanks for the clarification.

Reply:
Actually, these mutations are redundant because we already have upgrade path. However, it seems the upgrade path might not work properly in the rare cases, so the mutations are more for resilience.

Reply:
I feel it seems like an atomic problem which means something is not finished in the middle but it has been regarded as upgrade finished somehow. Do you think we need to create an issue to follow up to make the upgrade more solid?

Reply:
I read the upgrade path codes but cannot have any idea.
Yes, I can create a ticket for it and see how to make the path more solid.

Reply:
Sounds good. We can let another member help with that later.